app/views/shared/metadata/edit/_request.html.erb

```diff
@@ -5,7 +5,7 @@
     <%- request.request_metadata.field_infos.each do |field_info| %>
       <!-- todo unify with how submission use field info-->
       <%- if field_info.selection %>
-        <%= metadata_fields.select(field_info.key, field_info.selection) %>
+        <%= metadata_fields.select(field_info.key, field_info.selection, { prompt: "Select a #{field_info.key.to_s.gsub( '_', ' ')}"  }) %>
```

Comment:
I meant to comment on this one. Doesn't this change the default values of all fields? The story is asking for request type, read length and flowcell.